src/Capgemini.Xrm.DataMigration.FileStore/DataStore/DataFileStoreWriter.cs

```diff
@@ -98,6 +100,29 @@ public void SaveBatchDataToStore(List<EntityWrapper> entities)
             logger.LogVerbose("DataFileStoreWriter SaveBatchDataToStore finished");
         }
 
+        /// <summary>
+        /// Remove the value from the name property of any EntityReference type if obfuscation is being performed on the data
+        /// </summary>
+        /// <param name="entitiesToExport">Collection of entities being exported.</param>
+        public void RemoveEntityReferenceNameProperty(List<CrmEntityStore> entitiesToExport)
+        {
+            entitiesToExport.ThrowArgumentNullExceptionIfNull(nameof(entitiesToExport));
+
+            if (this.fieldsToObfuscate != null && this.fieldsToObfuscate.Any())
+            {
+                foreach (var entity in entitiesToExport)
+                {
+                    if (entity.Attributes.Where(x => x.AttributeType == "Microsoft.Xrm.Sdk.EntityReference").Any())
```

Comment:
Think the `if` statement here is redundant

Reply:
Agreed.